### DISCUSSION TITLE
Update links when note names change

### DISCUSSION THREAD

**Discussion by retospect** [SkepticMystic/breadcrumbs#449] [Q&A] [answered]
I am using dendrite naming conventions with the occasional extra up: [], and it is awesome. 

What I am looking for is a way to keep it from going brittle; when I change a linked name (or a dendrite prefix), all the links go stale, so it all becomes pretty rigid. Any suggestions

(Dendrite naming:
If notes are named like ```chicken```, ```chicken.foot``` and ```chicken.head```, ```chicken.foot``` automatically knows that its up node ```chicken```).

**Comment by icytree** [comment, marked as answer]
This plugin named [Obsidian Structured Plugin](https://github.com/dobrovolsky/obsidain-structure) should do the trick. But note that I noticed [a context where it would not work as expected](https://github.com/dobrovolsky/obsidain-structure/issues/14) (especially if you use a lot folder notes).

**Comment by retospect** [comment]
Thanks! That's really useful. I realized though that this is maybe a weird lonely path to be on. 
I changed to directories like most other people, and it seems it's all a bit better set up for that. 
(Migration with Bulk Rename and Folder Index plugins is not too bad).

**Comment by michaelpporter** [comment]
Closing as resolved (answered above). Reopen if it still comes up in v4.
